### ISSUE TITLE
Google Home Media Player

### ISSUE THREAD

**Issue by alexmc1510** [smartHomeHub/SmartIR#247]
**Home Assistant version** 0.101.2

**SmartIR version** 1.7.3

**SmartIR configuration** Samsung Media Player

**Describe the bug**

Media player for Samsung TV is working as expected using RMLink Pro+ but when I load configuration to Google Home Assistant to use Google Home Speaker I am only able to power on and off the TV. Other features are not available meanwhile plugin is fully working through Hassio.

**Comment by smartHomeHub** [comment]
I suppose that Google Home supports only on/off for media players

**Comment by claudiosala80** [comment]
I have the same issue.
Google home support up and down of volume for example.

For me was working about one month ago. After stop and there were errors in home assistant log calling volume, but never time to investigate.

Yesterday I updated home assistant and this add-on and google tell me "I cannot find device _mythings_"
I don't know exactly which is the device name is looking for.
I already removed and re-added home assistant integration in Google Home
